@anonklub/proof/package.json

```diff
@@ -19,7 +19,6 @@
     "privacy"
   ],
   "license": "MIT",
-  "files": ["dist"],
```

Comment:
we must not delete this, otherwise the ui won't be able to find the `@anonklub/proof` module during the build step. Especially in the GH action.

cc @0xisk https://github.com/anonklub/anonklub/actions/runs/7697466293/job/20975263017#step:12:2321